integration/helpers_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either expequals or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package integration provides FerretDB integration tests.
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func TestAssertEqualAltWriteError(t *testing.T) {
```

Comment:
I think test assertions should be simple enough so we don't have to write tests for tests. Otherwise, we will have to write tests for tests for tests.

Reply:
> I think test assertions should be simple enough so we don't have to write tests for tests. Otherwise, we will have to write tests for tests for tests.

I don't mind writing tests for test functions. I want to feel I wrote a function that works as expected and to feel calm with tests proving it.

Reply:
But how do you prove that tests work as expected? Will you write tests for tests?
Let's make this test helper so simple, so we will not need tests for it

Reply:
This test function has been removed.